lib/routes.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+
+import 'package:learning_compass_exp/screens/home/index.dart';
+import 'package:learning_compass_exp/screens/info/index.dart';
+
+import 'package:learning_compass_exp/data/models/petal_names.dart';
+
+final routes = <String, WidgetBuilder>{
+  '/': (context) => HomeScreen(),
+  '/income': (context) => InfoScreen(
+        name: PetalName.income,
```

Comment:
We could pass more data to the InfoScreen here as well. We could have the icons (if needed) and colors etc passed here. But that might lead to more duplicating....

Reply:
Depending on how we are going to handle the Petal data storing (in json or a static variable etc) passing variables here is an option. The InfoScreen will be needing probably at least the color and a String that contains the name (e.g. Income), and the PetalName of course.